src/packs/core_macros/Scan__Journal__z0pdvwE0FitMzwRJ.yml

```diff
@@ -364,33 +365,38 @@ command: >-
     //This checks and updates the scan entry for the target(s) if a single scan entry exists in the specified folder for the target(s) along with the updateExisting flag.
     //If either are false then this creates a new scan entry.
 
-    let scanObj = {};
-
-    scanObj.folder = journalFolder.id;
-
     let scanEntry;
 
-    let matchingJournalEntries = journalFolder.contents.filter(e => e.name.match(actor.name));
```

Comment:
`match` treats the `actor.name` as a regular expression. This causes naming methods that use parentheses to fail, and causes syntax errors if the actor name isn't a valid regular expression.